index.html

```diff
@@ -31,9 +31,144 @@ <h1 class="introduction"> Hey there. I’m Chukwuemeka <br> <span> I’m a softw
+                                </ul>        
+                                <button class="resumebutton buttons" type="submit"> See Project</button>        
+        
+                            </article>
+                </article>
+                <article id="recentcard" class="fifthbox">   
+                        <article class="innerdownbox">
+                                <p> Multi-Post Stories<br> Gain+Glory </p>
+                                <ul>
+                                        <li><a href="#">Ruby on rails</a></li>
+                                        <li><a href="#">css</a></li>
+                                        <li><a href="#">Javascript</a></li>
+                                        <li><a href="#">html</a></li>
+                                </ul>        
+                                <button class="resumebutton buttons" type="submit"> See Project</button>        
+        
+                            </article>
+
+                </article>
+                <article id="recentcard" class="sixthbox">
+                        <article class="innerdownbox">
+                                <p> Multi-Post Stories<br> Gain+Glory </p>
+                                <ul>
+                                        <li><a href="#">Ruby on rails</a></li>
+                                        <li><a href="#">css</a></li>
+                                        <li><a href="#">Javascript</a></li>
+                                        <li><a href="#">html</a></li>
+                                </ul>        
+                                <button class="resumebutton buttons" type="submit"> See Project</button>        
+        
+                            </article>
+                </article>  
+            </section>
+            
+ <!-- About me Section            -->
+
+            <article class="aboutme">
+                    <h2> About me</h2>
+                    <p class="describeme"> I can help you build a product ,<br> feature or website Look through <br>
+                         some of my work and experience! If <br> you like what you see and have a<br>
+                          project you need coded, <br>
+                            don’t hestiate to contact me.</p>
+                            <button class="resumebutton buttons" type="submit">Get My Resume</button>
+            </article>
+            <section class="grid-container"> 
+                    <article class="aboutmebox">
+                        <img src="img/languages.png" alt="language details">
+                        <p class="describeaboutmebox">Languages</p>
+                        <ul>
+                                <li><a href="#">Javascript</a></li>
+                                <li><a href="#">Ruby</a></li>
+                                <li><a href="#">Html</a></li>
+                                <li><a href="#">CSS</a></li>
```

Comment:
- [ ] I think it's better to use the family font as in the Figma template as required by the previous reviewer. Creates a better UI.